### ISSUE TITLE
Loss not decreasing

### ISSUE THREAD

**Issue by flaushi** [jcjohnson/fast-neural-style#147]
Hi,
I am trying to train an own model using an own style image. I have downloaded and created a h5 image set from MS COCO. Now I have taken a painting, and run train.lua with default paramters (I had to decrease batch size to 2 and style image size to 256 for limited memory reasons). However the results are not satisfying, and I observe my loss is not decreasing. I have also tried to decrease the learning rate to 0.0001, with no improvement (up to now😂). Any hints?
Thanks in advance!
